app/src/main/java/com/example/fooding/fragments/PantryFragment.java

```diff
@@ -116,4 +130,40 @@ public void onSuggestionClicked(IngredientSearchSuggestion suggestion) {
 
         recipeSearchView.setQuery("", false);
     }
+
+    ItemTouchHelper.SimpleCallback itemTouchHelperCallback = new ItemTouchHelper.SimpleCallback(0, ItemTouchHelper.LEFT) {
+        @Override
+        public boolean onMove(@NonNull RecyclerView recyclerView, @NonNull RecyclerView.ViewHolder viewHolder, @NonNull RecyclerView.ViewHolder target) {
+            return false;
+        }
+
+        @Override
+        public void onSwiped(@NonNull RecyclerView.ViewHolder viewHolder, int direction) {
+            selectedSuggestions.remove(viewHolder.getAdapterPosition());
+            suggestionsAdapter.notifyDataSetChanged();
+            suggestionsAdapter.notifyItemRemoved(viewHolder.getAdapterPosition());
+            Toast.makeText(getContext(), "Ingredient deleted", Toast.LENGTH_SHORT).show();
+
+            
+
+            queryIngredients();
+        }
+
+        private void queryIngredients() {
+            ParseQuery<Ingredient> query = ParseQuery.getQuery(Ingredient.class);
+            query.include(Ingredient.USER_KEY);
+            query.findInBackground(new FindCallback<Ingredient>() {
+                @Override
+                public void done(List<Ingredient> ingredients, ParseException e) {
+                    if (e != null) {
+                       Log.e(TAG, "Issue with getting ingredients", e) ;
+                       return;
+                    }
+                    for (Ingredient ingredient: ingredients) {
+
+                    }
```

Comment:
Oops looks like this is an empty loop left in. You could remove it, add the implementation for it, or have the implementation be part of a subsequent PR.

Reply:
Thanks, I removed that already too.